orchagent/portsorch.cpp

```diff
@@ -3130,6 +3139,36 @@ void PortsOrch::doTask(Consumer &consumer)
     }
 }
 
+bool PortsOrch::flushFdbEntries(Port port)
```

Comment:
As we discussed, we can use `fdborch->flushFDBEntries` instead

Reply:
addressed in next commit